deploy/crs/kieapp_trial_sso.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: app.kiegroup.org/v1
+kind: KieApp
+metadata:
+  name: trial
+spec:
+  environment: trial
+  auth:
+    type: sso
+    sso:
+      url: https://sso.example.com
+      realm: rhpam
+      adminUser: admin
+      adminPassword: example
+      clients:
+        console:
+          name: rhpamcentr-client
+          secret: somePwd
+        servers:
+          - name: kieserver-client-a
+            secret: otherPwd
+            hostnameHTTPS: kieserver-a.example.com
```

Comment:
what's this used for? is it required to specify the route here?

Reply:
This will be used by the EAP startup scripts to create a RHSSO Client in the realm with this redirectURL. If not set, the route hostname will be used instead (only when the client is set)

Reply:
ok, so its optional? if so, should we include it in our example repo? Seems like this would break in most clusters unless they're resolving example.com locally?

Reply:
This was just an example configuration. I better remove the file completely

Reply:
idk, might be good to have, but maybe just include the required fields? the user would see the optional fields post deployment... right?

Reply:
Moved to the README.md file